scripts/ci/k3d.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+# Install k3d if not found
+if ! command -v k3d &> /dev/null
+then
+    curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
+fi
+
+# CLUSTER_NAME is branch, or tag, fallback to ref
+CLUSTER_NAME=$(git symbolic-ref -q --short HEAD 2>/dev/null || git describe --tags --exact-match --short HEAD 2>/dev/null || git describe --all | tr "/" -)
```

Comment:
could you please add something like `fuseml-` prefix to the cluster name? So there's a distinction from possible other existing clusters...

Reply:
done.